.ci/generate

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
What is this script? Why do we need it? Isn't it the same like `make generate`?

Reply:
The script is intended to be used as step command for an optional `generate` concourse pipeline. It can be added to the
pipeline definition in forked repository to be able to update a branch with the uptodate generated files.

In my dev environment the generation of the controller-registration created something that does not match the version created by the verify step in central builds. For such cases it would be helpful to be able to execute and update the generation in the central build, also.

Reply:
Which version of helm are you using? helm 3 generates a different controller-registration than helm 2.
You can run `make install-requirements` to install the same version used in the pipeline, so you won't run into any verification issues.

Also, I don't think it's a good idea to run `hack/generate.sh` in the pipeline (especially not with `./...` which includes `vendor/...`), as `hack/check-generate.sh` will then not be able to detect, if generated files are not up-to-date and generated changes are simply ignored, which in turn might lead to undetected bugs in the committed code.
I see, you only want to execute it in your forked repository's pipeline. But then, I don't get, why we would need to add it here in g/g.

Reply:
It's not a problem of `make install-requirements`. I've called it multiple times, it does not make a difference.

The vendor folder is ignored for go package pats. I always call it this way for my projects. But Rafael already proposed just to call `make generate`. I've changed this.

Reply:
I am not completely against adding the file here, but I think, there might be better options.
For example, you might look into GitHub Actions (which is quite easy to setup) to run generation jobs on your fork. There you have every freedom of configuring jobs and run steps as you wish, so you/we don't have to add a `.ci` script. Bonus: you can even let Actions directly commit to your fork very easily, if you want 😉

Reply:
I've removed the file again.

Reply:
Hmm, it's still here 🤔

Reply:
Hmm, I don't see it anymore